example/spec/parallel/ingestGranule/IngestGranuleFtpSuccessSpec.js

```diff
@@ -41,6 +43,10 @@ describe('The FTP Ingest Granules workflow', () => {
     console.log('\nStarting ingest test');
     inputPayload = JSON.parse(fs.readFileSync(inputPayloadFilename, 'utf8'));
     inputPayload.granules[0].dataType += testSuffix;
+    inputPayload.granules[0].granuleId = randomStringFromRegex(granuleRegex);
```

Comment:
idk if it makes sense to use this here, but there's a `setupTestGranuleForIngest` function in `helpers/granuleUtils` that may be worth looking at...?

Reply:
https://github.com/nasa/cumulus/blob/master/example/spec/helpers/granuleUtils.js#L47

Reply:
nvm - looking further into it this really wouldn't offer any benefit to what you're currently doing. Disregard 😄

Reply:
Yeah, I looked at that, but the random granule function was the *only* thing it was doing we cared about.